packages/python-google-compute-engine/google_compute_engine/boto/boto_config.py

```diff
@@ -85,5 +86,7 @@ def _CreateConfig(self, project_id):
     config.SetOption('GSUtil', 'default_project_id', project_id)
     config.SetOption('GSUtil', 'default_api_version', '2')
     config.SetOption('GoogleCompute', 'service_account', 'default')
-    config.SetOption('Plugin', 'plugin_directory', boto_dir)
+    # Don't set the plugin for python3 as it is not compatible.
```

Comment:
The version of Python you're using to run *this script* doesn't necessarily correspond to the version of Python that users will invoke to run code that relies on Boto (gsutil, for example, can be run with either Python 2 or 3).

As mentioned in our Hangouts discussion, I think we can just stop writing this plugin line entirely.  Gsutil doesn't need it. If we have any documentation that instructs people to use Boto, we should also update it to mention that users should install the google_compute_engine plugin and set up their boto config file (using the lines below) such that Boto loads the GCE oauth2 auth_handler.

Reply:
SGTM